test/functionalTest/cases/117_convop_using_standard_ops/postContextEntityTypeAttribute.test

```diff
@@ -0,0 +1,170 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp postContextEntityTypeAttribute: POST /v1/registry/contextEntityTypes/{entityId::type}/attributes/A1
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Register entities of type ET using POST /v1/registry/contextEntityTypes/ET/attributes/A1 and payload
+# 02. Discover entities of type ET
+# 03. Update registration using POST /v1/registry/contextEntityTypes/ET/attributes/A1 and payload
+# 04. Discover entities of type ET
+#
```

Comment:
Because with this convop you can register an attribute. With the other two you cannot, and the attribut is needed for the forwarding.
In the other two I add the attribute after the first registration.

NTC